.rubocop.yml

```diff
@@ -75,4 +75,7 @@ Style/FrozenStringLiteralComment:
 Rails/Output:
   Exclude:
     - 'config/unicorn.rb'
-    - 'db/seeds.rb'
\ No newline at end of file
+    - 'db/seeds.rb'
+
+Style/EmptyMethod:
```

Comment:
You can add ignores for the few files that have this, using:
`bundle exec rubocop --auto-gen-config`